src/styles/utils/flexs/flexs.scss

```diff
@@ -0,0 +1,11 @@
+@mixin flex-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
logically it also mixin so or move it to mixin file directly or create separate file for all mixins and move it there